titus_isolate/event/kubernetes_opportunistic_window_publisher.py

```diff
@@ -51,41 +60,68 @@ def __get_node(self) -> V1Node:
         log.debug('node: %s', node)
         return node
 
+    def __watch(self):
+        label_selector = "{}={}".format(OPPORTUNISTIC_RESOURCE_NODE_NAME_LABEL_KEY,
+                                        self.__node_name)
+        while True:
+            log.info("Starting opportunistic resource watch...")
+            try:
+                stream = watch.Watch().stream(
+                    self.__custom_api.list_cluster_custom_object,
+                    "titus.netflix.com",
```

Comment:
with this argument list, I don't really know what these arguments are, and the definition isn't super helpful
https://github.com/kubernetes-client/python-base/blob/bf5c59987122fd35e9bb14a939f3db94e55b30bf/watch/watch.py#L102

Are there other keyword args you could use instead to help future-proof the ordering?

Reply:
yes, done.